src/main/java/frc/robot/subsystems/Climber2022.java

```diff
@@ -0,0 +1,216 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj.Relay;
+import edu.wpi.first.wpilibj.Relay.Value;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.button.NetworkButton;
+import frc.robot.RobotConstants;
+import frc.robot.commands.Climber2022EnableCMD;
+import frc.robot.commands.Climber2022DisableCMD;
+import frc.robot.commands.Climber2022SetLeftSpeed;
+import frc.robot.commands.Climber2022SetRightSpeed;
+import frc.robot.commands.Climber2022StopCMD;
+import frc.robot.logging.RobotLogManager;
+import frc.robot.motors.MotorControllerEncoder;
+import frc.robot.motors.MotorControllerFactory;
+import frc.robot.motors.MotorType;
+
+import frc.robot.tuning.SubsystemTuner;
+import java.util.Map;
+import org.apache.logging.log4j.Logger;
+
+public class Climber2022 extends SubsystemTuner {
+
+  // TODO make climber into state space and use real position control, after granite state
+  private final MotorControllerEncoder climberMotorLeft = MotorControllerFactory.create(
+      RobotConstants.get().climber2022LeftMotorId(), MotorType.SPARK_MAX_BRUSHLESS);
+  private final MotorControllerEncoder climberMotorRight = MotorControllerFactory.create(
+      RobotConstants.get().climber2022RightMotorId(), MotorType.SPARK_MAX_BRUSHLESS);
+  private final Relay climberLock = new Relay(RobotConstants.get().climber2022SolenoidChannel());
```

Comment:
these should not have their values filled instantly, but instead done in the constructor